app/views/devise/invitations/new.html.haml

```diff
@@ -3,5 +3,7 @@
   = simple_form_for resource, :as => resource_name, :url => invitation_path(resource_name), :html => {:method => :post} do |f|
     - resource.class.invite_key_fields.each do |field|
       = f.input field
+      = f.input :roles, as: :radio_buttons, collection: User.values_for_roles.map{ |r| [r, r.to_s.capitalize] }, label_method: :second, value_method: :first, checked: f.object.roles
```

Comment:
I think we could like this to simplify it:

```rb
f.input :roles, as: :radio_buttons, collection: User.values_for_roles, label_method: ->(r) { r.to_s.capitalize }, checked: f.object.roles
```

Later on when we use i18n, we might even don't need custom label method.